python/taichi/misc/constants.py

```diff
@@ -0,0 +1,19 @@
+import math
```

Comment:
Once again, I don't think these belong to this PR. Please revert.

Please also re-read https://taichi.readthedocs.io/en/latest/contributor_guide.html#making-good-pull-requests